tequilapi/endpoints/node.go

```diff
@@ -35,6 +35,9 @@ type nodeMonitoringAgent interface {
 	TransferredData(rangeTime string) (node.TransferredData, error)
 	SessionsCount(rangeTime string) (node.SessionsCount, error)
 	ConsumersCount(rangeTime string) (node.ConsumersCount, error)
+	SeriesEarnings(rangeTime string) (node.SeriesEarnings, error)
+	SeriesSessions(rangeTime string) (node.SeriesSessions, error)
```

Comment:
Try to keep the formatting consistent. 
If you have `SessionsCount` it should `SessionsSeries`.